section-parsers.go

```diff
@@ -123,6 +123,10 @@ func getDefaultParser() *Parsers {
 		&parsers.DefaultBackend{},
 		&parsers.UniqueIDFormat{},
 		&parsers.UniqueIDHeader{},
+
+		&simple.Option{Name: "splice-auto"},
+		&simple.Option{Name: "splice-request"},
+		&simple.Option{Name: "splice-response"},
```

Comment:
as you may notice all options are grouped together. 
so this also needs to be moved a bit up to be with other options.

since options are not in alphabetical or any order I leave up to you to decide where exactly :).

This is important since order of parsers in list is important (the same order is used to also write all the options in file)